### ISSUE TITLE
no mic noise reduction with skype

### ISSUE THREAD

**Issue by promeneur** [wwmm/easyeffects#1703]
openSUSE Tumbleweed
easyeffects 6.2.6

I do a Skype test call.
During my speech, I stroke my finger against my desktop. This adds "toc","toc",…,"toc".

When I listen to the record, i ear the tocs.

![pipewire](https://user-images.githubusercontent.com/240424/181720907-baa93bf0-3062-4a4e-8f75-24b1650f51fb.png)
![settings_1](https://user-images.githubusercontent.com/240424/181720958-b62f8f81-2974-4b96-ae3f-38b9dac9bd56.png)
![settings_2](https://user-images.githubusercontent.com/240424/181720985-481a9b37-aa49-4c81-8e38-093065e045c4.png)
![settings_3](https://user-images.githubusercontent.com/240424/181720999-d87a6bd6-6c95-4dc5-ba3d-05164f3c5a23.png)

**Comment by wwmm** [comment]
I don't think this can be fixed here. Our noise reduction filter uses the RNNoise library. And this library is based on a neural network that is trained to remove some kinds of noises. In order to filter custom types of noises like this one you will probably have to use the scripts at the rnnoise page and train a custom model for them.

This library is good but it doesn't filter by default any possible type of noise. There is some dependence on the signal/noise ratio too.

**Comment by wwmm** [comment]
So much has changed that is probably better to close this issue
